### PR TITLE
[release-next][test] Resolve testing configuration for sugar controller

### PR DESCRIPTION
**Pull request by pierDipi**
Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

### CODE REVIEW

File: openshift/release/generate-release.sh [hunk 1]
```diff
@@ -17,10 +17,8 @@ fi
 # the core parts
 resolve_resources config/ $output_file $image_prefix $tag
 
-# Sugar Controller
-resolve_resources config/sugar/ crd-sugar-resolved.yaml $image_prefix $tag
-cat crd-sugar-resolved.yaml >> $output_file
-rm crd-sugar-resolved.yaml
+# Sugar testing configuration
+cat test/config/sugar.yaml >> $output_file
```
Comment (matzew) [inline openshift/release/generate-release.sh:21]: do we really need this?

Comment (pierDipi) [reply openshift/release/generate-release.sh:21]: that's the fix, so yes :)

**Comment by pierDipi** [comment]
Ok, the fix works, this PR was just a test to prove that fix works, see https://github.com/openshift/knative-eventing/pull/1693

**Pull request closed without merging**